### ISSUE TITLE
Support for email settings

### ISSUE THREAD

**Issue by ktibi** [datadrivers/terraform-provider-nexus#355]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Can we have the posibility to configure email settings ? Or maybe an example with nexus_script ressource ?

### New or Affected Resource(s)/Data Source(s)

nexus_email

### Pro feature

- [ ] Yes. Pro license Required

### Potential Terraform Configuration

```HCL
ressource "nexus_email" "email" {
  host     = "smtp.example.Com"
  port     = "25"
  username = "user"
  password = "pass"
  enabled  = "true"
  ...
}
```

**Comment by ktibi** [comment]
thanks @fabrue :)

**Comment by anmoel** [comment]
@ktibi it is released in version 1.22.0 https://registry.terraform.io/providers/datadrivers/nexus/latest/docs/resources/mail_config
